sweep/sweeper.go

```diff
@@ -1666,7 +1700,7 @@ func (s *UtxoSweeper) handleBumpEventTxFailed(resp *bumpResp) error {
 	// TODO(yy): should we also remove the failed tx from db?
 	s.markInputsPublishFailed(resp.set)
 
-	return err
+	return nil
```

Comment:
is it temporary that we swallow the error here? ie, will we use the return value again in future?

Reply:
We are expecting an error here, so I think we will always return nil here, so probably better to remove the error return value at all.

Reply:
updated the commits to make it clear - we no longer need this error here